crates/aptos-telemetry-service/src/lib.rs

```diff
@@ -107,6 +109,14 @@ impl AptosTelemetryServiceArgs {
         let validator_fullnodes = Arc::new(aptos_infallible::RwLock::new(HashMap::new()));
         let public_fullnodes = config.pfn_allowlist.clone();
 
+        let downtime_metrics_cache = Arc::new(aptos_infallible::RwLock::new(VecDeque::<
+            MetricsEntry,
+        >::new()));
+        let downtime_metrics_cache_updater = DowntimeMetricsCacheUpdater::new(
+            downtime_metrics_cache.clone(),
+            Duration::from_secs(60000),
```

Comment:
Also would suggest extracting into a constant
```suggestion
            Duration::from_secs(60),
```